docs/source/ivr/ivr.rst

```diff
@@ -316,3 +316,11 @@ Custom exception
         }
 
     This will allow you to return an error code that your IVR provider can understand.
+
+Using IVR module with IVR providers
+===================================
```

Comment:
Since you have moved this file to a new directory, all its images do no longer work.
```/home/sebastian/modules/docs/source/ivr/ivr.rst:224: WARNING: image file not readable: ivr/img/ivr_initiate_call_task.png
/home/sebastian/modules/docs/source/ivr/ivr.rst:None: WARNING: image file not readable: ivr/img/ivr_settings.png
/home/sebastian/modules/docs/source/ivr/ivr.rst:None: WARNING: image file not readable: ivr/img/ivr_cdr.png```